### ISSUE TITLE
frontend: detailed view of a schema

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#95]
## User story
1. As a {HR-employee}
2. I want / need {to get a detailed view of a schema }
3. So that {I can see the important content of a schema (attributes) and see the schema's corresponding credential definitions }

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...
![grafik](https://user-images.githubusercontent.com/92971164/171156213-14cf4714-aaba-4011-898d-47c16d40fbae.png)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
